src/AI/CapPhaseAnalyzer.java

```diff
@@ -363,7 +363,7 @@ public void buildBaseCapChains(GameMap map, Army viewer, ArrayList<XYCoord> righ
           }
           madeProgress = true; // We have somewhere we can still get to
 
-          final int distance = infPath.getFuelCost(inf, map);
+          final int distance = infPath.getMoveCost(inf, map);
```

Comment:
What determines whether you use Fuel or Move cost? Other places (e.g. line 280) still use Fuel when it looks like Move might be more apropos.

Reply:
How tired I was, probably.